tests/test_run_process.py

```diff
@@ -87,9 +87,11 @@ async def test_async_alive_terminates(mocker):
     mock_start_process = mocker.patch('watchgod.main._start_process')
     mock_start_process.return_value = FakeProcess()
     mock_kill = mocker.patch('watchgod.main.os.kill')
-    f = Future()
-    f.set_result(1)
-    c = mocker.MagicMock(return_value=f)
+
+    async def f():
+        return 1
+
+    c = mocker.AsyncMock(return_value=f)
```

Comment:
I think you can just set the return value directly, and it will be returned on `await`.